src/test/scenarios/attestation/output/src/attestation/README.md

```diff
@@ -1,5 +1,32 @@
-Microsoft Azure CLI 'attestation' Extension
-==========================================
+# Azure CLI attestation Extension #
+This is a extension for attestation
 
-This package is for the 'attestation' extension.
-i.e. 'az attestation'
+### How to use ###
+Install this extension using the below CLI command
+```
+az extension add --name attestation
+```
+
+### Included Features ###
+#### attestation: ####
+attestation
+* Examples:*
+```
+az attestation create-provider --provider-name "myattestationprovider" --resource-group "MyResourceGroup"
+
+az attestation list-operation
+
+```
+#### attestation attestation-provider: ####
+attestation attestation-provider
+* Examples:*
```

Comment:
Is there any description for each command/operation when auto-ghen?

Reply:
Then, the template may need change. add more level title for command. such as?
#### attestation attestation-provider: ####
##### attestation attestation-provider show: #####
##### attestation attestation-provider update: #####